mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -785,6 +913,7 @@
         <Effect index="0" value="LocustMPCreateSet"/>
         <Flags index="BestUnit" value="0"/>
         <Flags index="RequireTargetVision" value="0"/>
+        <Flags index="Transient" value="1"/>
```

Comment:
**undocmented**: Spawning Locusts with a target point is now a transient ability. The Locust will not lose existing orders when the player uses this